File: trunk/xMBSv1/CodeFirstMembershipSharp/Context/DataContext.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Data.Entity;

namespace CodeFirstMembershipSharp
{
    public class DataContext : DbContext
    {
        public DataContext() : base("xMBSv1") { }

        public DbSet<User> Users { get; set; }
        public DbSet<Role> Roles { get; set; }
    }
}

File: xMBSv1/CodeFirstMembership/Context/DataContext.vb
Imports System.Data.Entity

Public Class DataContext
    Inherits DbContext

    'Protected Overrides Sub OnModelCreating(modelBuilder As DbModelBuilder)
    '    modelBuilder.Conventions.Remove(Of Infrastructure.IncludeMetadataConvention)()
    'End Sub

    Public Property Users As DbSet(Of User)
    Public Property Roles As DbSet(Of Role)

End Class